Src/SwqlStudio/DocumentationContent.Designer.cs

```diff
@@ -62,7 +62,6 @@ private void InitializeComponent()
             this.Controls.Add(this.docTextBox);
             this.Controls.Add(this.itemTypeLabel);
             this.Name = "DocumentationContent";
-            this.Padding = new System.Windows.Forms.Padding(5);
```

Comment:
For some reason, padding inside `WeifenLuo.WinFormsUI.Docking.DockContent` completely breaks layout in HiDPI. While padding from left works fine, when padding from the remaining sides, 5 pixels are rendered as 50. It was simper to just remove it and adjust the locations of `itemTypeLabel` and `docTextBox` accordingly.